app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Admin
+  class CategoriesController < ApplicationController
+
+    def index
+      @categories = Category.all.where(parent_category_id: nil)
```

Comment:
use scope `root` ?

Reply:
it automatically handles when merged with main as these changes is made in previous branches.